ui/components/multichain/account-list-menu/index.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { AccountListMenu } from '.';
+
+export default {
+  title: 'Components/Multichain/AccountListMenu',
+  component: AccountListMenu,
+  argTypes: {
+    onClose: {
+      action: 'onClose',
+    },
+  },
+};
+
+export const DefaultStory = (args) => <AccountListMenu {...args} />;
```

Comment:
Getting this error in the story
https://user-images.githubusercontent.com/39872794/225660012-2881947d-c993-4ee1-a9ae-ad0c4c1e3c75.mov